Library/Homebrew/extend/ARGV.rb

```diff
@@ -88,8 +88,9 @@ def next
   end
 
   def value(arg)
-    arg = find { |o| o =~ /--#{arg}=(.+)/ }
-    $1 if arg
+    return if arg.empty? || arg.index("-") || arg.index("=")
```

Comment:
I'm not sure whether this should raise an exception instead of silently ignoring an invalid `arg`. Feedback welcome!